plugins/lodash.js

```diff
@@ -0,0 +1,3 @@
+const LodashModuleReplacementPlugin = require('lodash-webpack-plugin');
```

Comment:
I am not sure how useful this file is.

Can we do
```js
module.exports = require('lodash-webpack-plugin')
```
?

Reply:
We could. I was just trying to keep consistent with the file structure of the other plugins. Let's discuss.

Reply:
I see, that is reasonable.